pkg/virt-handler/vm.go

```diff
@@ -786,6 +793,33 @@ func (d *VirtualMachineController) heartBeat(interval time.Duration, stopCh chan
 	}
 }
 
+func (d *VirtualMachineController) addNodeCpuManagerLabel() {
+	entries, err := filepath.Glob("/proc/*/cmdline")
+	if err != nil {
+		log.DefaultLogger().Reason(err).Errorf("failed to set a cpu manager label on host %s", d.host)
+		return
+	}
+
+	for _, entry := range entries {
+		content, err := ioutil.ReadFile(entry)
+		if err != nil {
+			log.DefaultLogger().Reason(err).Errorf("failed to set a cpu manager label on host %s", d.host)
+			return
+		}
+
+		if strings.Contains(string(content), "kubelet") && strings.Contains(string(content), "cpu-manager-policy=static") {
```

Comment:
The kubelet can also be configured via a configfile with --config now. It's possible this cpu manager value won't be in the comandline.

https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/#start-a-kubelet-process-configured-via-the-config-file

Reply:
Maybe, I'm not sure. 
However, in the worst case, this node won't be labeled and no harm will be done.
But I've looked at openshift and they also add the values to the command line, while the configuration of the cpu manager is in a configmap.

In general, we will need to get a more deterministic answer where cpu manager is running or not. I've opened a bug https://github.com/kubernetes/kubernetes/issues/66525

I don't see a risk here. It's not perfect and we should improve it.